ui/admin/app/components/form/worker/create-worker-led/index.js

```diff
@@ -35,18 +52,39 @@ worker {
             .map((tag) => `"${tag?.trim()}"`)
             .join(', ')
         : '"<tag1>", "<tag2>"'
-    }]
+    }],
+  },
+  ${
+    this.features.isEnabled('byow-pki-upstream')
+      ? `initial_upstreams = [${
+          this.initialUpstreams
+            ? this.initialUpstreams
+                .split(',')
+                .filter((tag) => tag.trim())
+                .map((tag) => `"${tag?.trim()}"`)
+                .join(', ')
+            : '\n\t"<upstream1>",\n\t"<upstream2>",\n\t"<upstream3>"\n'
```

Comment:
We shouldn't use tabs here since we use spaces everywhere else and users are going to be copying this config.